### PR TITLE
Run docker with the same uid as tester user

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

**Comment by sileht** [comment]
Pull-request updated, HEAD is now dd33ed02795e0023067295c80055d15095af61e1

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 237e2b75ccee9b507dc49ec93a6d2fb98f704d1c

**Comment by jd** [comment]
I think you need "" because of the :

File: .travis.yml [hunk 1]
```diff
@@ -35,11 +35,10 @@ before_script:
...
 install:
   - docker pull gnocchixyz/ci-tools:latest
-  # Allow docker to write in home directory
-  - chmod -R o+w ~/
-  - find ~/ -type d -exec chmod o+x {} \;
+  - chown -R 1000: .
+  - chown -R 1000: ~/.cache/pip
 script:
-  - docker run -v ~/.cache/pip:/home/tester/.cache/pip -v $(pwd):/home/tester/src gnocchixyz/ci-tools:latest tox -e ${TARGET}
+  - docker run --user 1000 -v ~/.cache/pip:/home/tester/.cache/pip -v $(pwd):/home/tester/src gnocchixyz/ci-tools:latest tox -e ${TARGET}
```
Comment (jd) [inline .travis.yml]: --user $UID

and do not use chown?

**Comment by sileht** [comment]
Not needed anymore

**Pull request closed without merging**